src/config/settings/base.py

```diff
@@ -256,6 +255,13 @@
         }
     },
     "root": {"level": "INFO", "handlers": ["console"]},
+    # "loggers": {
+    #     "rules": {
+    #         "handlers": ["console"],
+    #         "level": "DEBUG",
+    #         "propagate": True,
+    #     },
+    # },
```

Comment:
I can remove this if folks prefer-- it's the logging config for seeing which permissions are being evaluated